app/controllers/trainees/personal_details_controller.rb

```diff
@@ -34,5 +37,16 @@ def personal_details_params
         nationality_ids: [],
       )
     end
+
+    def redirect_to_confirm
+      redirect_to(trainee_personal_details_confirm_path(trainee))
+    end
+
+    def section_completed?
+      ProgressService.call(
+        validator: PersonalDetail.new(trainee),
+        progress_value: trainee.progress.personal_details,
+      ) == Progress::STATUSES[:completed]
```

Comment:
```suggestion
      ProgressService.call(
        validator: PersonalDetail.new(trainee),
        progress_value: trainee.progress.personal_details,
      ).completed?

```

can we not do something like that?

Reply:
I like. Done in https://github.com/DFE-Digital/register-trainee-teachers/pull/161/commits/0ed3bc1d2e73aecce233ca59c5d5e5683429a6f9